tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_then_reset.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.clock.app import Clock
+
+class TestClockRunStopWatch(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+
+        self.clock = Clock(self.marionette)
+        self.clock.launch()
+
+    def test_click_run_stopwatch_then_reset(self):
+
+        stopwatch_view = self.clock.switch_view("stopwatch")
+
+        self.assertEqual(stopwatch_view.current_time, '00:00.00')
+
+        stopwatch_view.tap_start()
+
+        self.assertNotEqual(stopwatch_view.current_time, '00:00.00')
+
+        stopwatch_view.tap_pause()
+
+        self.assertNotEqual(stopwatch_view.current_time, '00:00.00')
+        self.assertTrue(stopwatch_view.verify_laps_not_same())
```

Comment:
I tried something like this:
        time1 = stopwatch_view.current_time
        time.sleep(2)
        self.assertEqual(time1, stopwatch_view.current_time, "both times not the same")

And that works on desktop and device. I don't think this would warrant a new method.